engine/consensus/sealing/engine.go

```diff
@@ -50,7 +50,7 @@ type Engine struct {
 	receiptSink                          EventSink
 	approvalSink                         EventSink
 	requestedApprovalSink                EventSink
-	pendingReceipts                      *fifoqueue.FifoQueue
+	pendingReceipts                      *fifoqueue.FifoQueue // TODO replace with engine.FifoMessageStore
```

Comment:
Since this PR has been big, I left this for future refactor

Reply:
💯 Yurii is anyway working on the sealing engine. No need to update code that is undergoing major refactoring.